test/inductor/test_torchinductor.py

```diff
@@ -15456,7 +15456,14 @@ def fn(pytype, dtype):
                         ret_opt = fn_opt(pytype, dtype)
 
                 self.assertEqual(ret_opt, fn(pytype, dtype))
-
+def test_to_dtype_use_compute_types():
```

Comment:
The indent looks weird. Is this a CPU-only test or not? Is it a part of `TestFull`?